parsec/core/fs/remote_loader.py

```diff
@@ -597,11 +597,15 @@ def __init__(self, remote_loader: RemoteLoader, timestamp: Pendulum):
     async def load_manifest(
         self, entry_id: EntryID, version: int = None, timestamp: Pendulum = None
     ) -> RemoteManifest:
-        if timestamp is not None and timestamp != self.timestamp:
-            raise FSError(
-                f"Cannot load a manifest at a different timestamp through a timestamped remote loader"
-            )
-        return await super().load_manifest(entry_id, version=version, timestamp=self.timestamp)
+        return await super().load_manifest(
+            entry_id,
+            version=version,
+            timestamp=None
+            if version is not None
+            else self.timestamp
+            if timestamp is None
+            else timestamp,
+        )
```

Comment:
A comment explaining why a timestamped remote loader might need to load manifest from yet another timestamp?

Reply:
Done!